src/content.test.ts

```diff
@@ -20,7 +20,7 @@ describe("content", () => {
         store = new TestStore();
 
         setConfig({
-          currentFromId: "dsnp://0x0123456789ABCDEF",
+          currentFromId: "0x0123456789ABCDEF",
```

Comment:
Since the user is going to have to set `currentFromId` manually, why don't we just change it to `currentFromURI` and store a full DSNPUserURI instead so the user never has to convert between URIs and Ids?

Reply:
fixed